### PR TITLE
Add instructions how to setup a new customer.

### PR DESCRIPTION
**Pull request by haustre**
### Contributor checklist

- [x] My contribution is **ready to be merged** as is
- [x] I verified the modifications **render properly**
- [x] I used **spell checking**
- [x] I **used the styling and structure aids available** in [Sphinx/ResT](http://www.sphinx-doc.org/en/stable/rest.html). (Links use `` `…`_``, enumerations `#.`, lists `*`, code ``` ``…`` ```/`.. code::`, warnings .. `warning::`, etc.)
- [x] I reread the text keeping in mind that the text has to be **comprehended** fully **by the target audience**

### CODE REVIEW

File: devops/business_services/new_customer.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Postgres Database
+^^^^^^^^^^^^^^^^^
+
+Add the new databases to the `VSHN git Repository`_.
+The database will automatically be created after a few minutes.
+
+.. _VSHN Git Repository: https://git.vshn.net/tocco/tocco_hieradata/edit/master/database/master.yaml
+
+| Use the same password for both databases of the installation
+| This password is also used to setup OpenShift.
```
Comment (pgerber) [inline devops/business_services/new_customer.rst]: > This password is also used to setup OpenShift.

I'd be a bit more specific, like "This password needs to be used as `DB_PASS` parameter to `oc process` later on."

File: devops/business_services/new_customer.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+| Generate a db password:
+
+.. code-block:: bash
+
+  pwgen -s 20 1
+
+Main Database:
+
+.. code-block:: text
```
Comment (pgerber) [inline devops/business_services/new_customer.rst]: ~~text~~**yaml**

File: devops/business_services/new_customer.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+  nice_${CUSTOMER}:
+    db_user: 'nice_${CUSTOMER}'
+    db_password: '${DB_PASS}'
+    extensions:
+      - 'lo'
+      - 'uuid-ossp'
+
+
+History Database (only version 2.19 and higher):
```
Comment (pgerber) [inline devops/business_services/new_customer.rst]: Do we still setup anything that's older than 2.19?

File: devops/business_services/new_customer.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+.. code-block:: text
+
+  nice_${CUSTOMER}_history:
```
Comment (pgerber) [inline devops/business_services/new_customer.rst]: ~~text~~**yaml**

File: devops/business_services/new_customer.rst [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    db_user: 'nice_${CUSTOMER}'
+    db_password: '${DB_PASS}'
+    extensions:
+      - 'lo'
```
Comment (pgerber) [inline devops/business_services/new_customer.rst]: ~~`extensions: [ lo ] `~~

File: framework/architecture/s3/properties.rst [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Properties file
```
Comment (pgerber) [inline framework/architecture/s3/properties.rst]: ~~f~~**F**ile

File: devops/business_services/index.rst [hunk 7]
```diff
@@ -0,0 +1,8 @@
+Business services
```
Comment (pgerber) [inline devops/business_services/index.rst]: ~~s~~**S**ervices

File: devops/business_services/new_customer.rst [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+--------------------
+
+:ref:`new-installation-openshift`
+
+Create in TeamCity
+-------------------
+
+:ref:`new-installation-cd`
+
+Final stepes
```
Comment (pgerber) [inline devops/business_services/new_customer.rst]: ~~s~~**S**teps

File: framework/architecture/s3/properties.rst [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+===============
+
+Introduction
+------------
+
+The S3 module is available from version 2.19.
+
+Connect to database over ssh
```
Comment (pgerber) [inline framework/architecture/s3/properties.rst]: > Connect to database over ssh
> ----------------------------
> 
> The S3 storage is accessible publicly.
> But if you change the "dataSource.serverName" in the "hikaricp.properties" to localhost you have to adjust the "s3.checksum" property.
> The checksum for use with localhost is already in the s3.properties file but commented out. It just needs to be commented in.

Hasn't the checksum been removed?

File: framework/configuration/new_customer.rst [hunk 10]
```diff
@@ -0,0 +1,35 @@
+.. _add_customer_module:
+
+Add new Customer module
```
Comment (pgerber) [inline framework/configuration/new_customer.rst]: ~~m~~**M**odule

File: framework/configuration/new_customer.rst [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+  .. figure:: new_customer/modules.png
+        :scale: 60%
+
+ 4. Check the generated dependencies, there are probably some mistakes in there. Add the dependencies to the customers pom
+
+S3 module
+^^^^^^^^^
+
+Add this entry to the customer pom **if the customer should use S3**:
```
Comment (pgerber) [inline framework/configuration/new_customer.rst]: > if the customer should use S3

Shouldn't S3 be enabled for all new customers? If not, can you clarify when it has to be enabled.

File: devops/business_services/new_customer.rst (hunk 5 above)
Comment (pgerber) [reply devops/business_services/new_customer.rst]: I actually meant it should be removed for the history DB.

File: framework/configuration/new_customer.rst (hunk 11 above)
Comment (pgerber) [reply framework/configuration/new_customer.rst]: Hey, you ignored my comment :(.

File: framework/configuration/new_customer.rst (hunk 10 above)
Comment (pgerber) [reply framework/configuration/new_customer.rst]: I feel ignored :(.

File: framework/configuration/new_customer.rst (hunk 11 above)
Comment (pgerber) [reply framework/configuration/new_customer.rst]: What customers shouldn't use it?

Comment (haustre) [reply framework/configuration/new_customer.rst]: At the moment no customer should use it. But in a few weeks all of them should.

**Pull request merged**